index.js

```diff
@@ -8,6 +8,7 @@ function isError(obj) { return toS(obj) === '[object Error]'; }
 function isBoolean(obj) { return toS(obj) === '[object Boolean]'; }
 function isNumber(obj) { return toS(obj) === '[object Number]'; }
 function isString(obj) { return toS(obj) === '[object String]'; }
+function isBuffer(obj) { return typeof Buffer !== 'undefined' && typeof Buffer.isBuffer === 'function' && Buffer.isBuffer(obj); }
```

Comment:
I think for bundler reasons, we need to use this: https://github.com/inspect-js/node-deep-equal/blob/master/index.js#L166-L178

Reply:
Ok, I've updated the code. Or did you mean to add a new dependency?

Reply:
no, that code doesn't exist as a separate package yet, and `is-buffer` isn't exactly the same. later, i'll look into whether is-buffer works, or whether i'd need to make a PR or a new package, and then i'd add the dep to both deep-equal and here.